generator/feature_segments_checker/feature_segments_checker.cpp

```diff
@@ -181,14 +181,14 @@ class Processor
       m_uniqueRoadPoints.insert(RoughPoint(f.GetPoint(i)));
 
     // Preparing feature altitude and length.
-    TAltitudeVec pointAltitudes(numPoints);
+    TAltitudes pointAltitudes(numPoints);
     vector<double> pointDists(numPoints);
     double distFromStartMeters = 0;
     for (uint32_t i = 0; i < numPoints; ++i)
     {
       // Feature segment altitude.
       TAltitude altitude = m_srtmManager.GetHeight(MercatorBounds::ToLatLon(f.GetPoint(i)));
-      pointAltitudes[i] = altitude == generator::SrtmTile::kInvalidHeight ? 0 : altitude;
+      pointAltitudes[i] = altitude == kInvalidAltitude ? 0 : altitude;
```

Comment:
Why not to keep kInvalidAltitudes as is? Your tool will calculate number of kInvalidAltitudes.